.golangci.yml

```diff
@@ -0,0 +1,150 @@
+# options for analysis running
```

Comment:
Where did this version of the lint config come from?

In general, I would like to see this converge with containerd's configuration, see https://github.com/containerd/containerd/blob/master/.golangci.yml

If there are changes here I would like to see them applied in both places

Reply:
Oh this one is coming from `example` config from golangci-lint.

However, it's making sense to keep the same as other repo (e.g. containerd). Let me make an update.